### PR TITLE
chore: add dynamic wrapper for nexus-repository-manager plugin

### PR DESCRIPTION
**Pull request by JessicaJHee**
## Description

Adds dynamic wrapper for nexus-repository-manager plugin after its migration to the community-plugins repo.

## Which issue(s) does this PR fix

- Fixes [RHIDP-3803](https://issues.redhat.com/browse/RHIDP-3803)

Please make sure that the following steps are complete:

- [ ] GitHub Actions are completed and successful
- [ ] Unit Tests are updated and passing
- [ ] E2E Tests are updated and passing
- [ ] Documentation is updated if necessary (requirement for new features)
- [ ] Add a screenshot if the change is UX/UI related

### CODE REVIEW

**Comment by 04kash** [comment]
The e2e-tests seem to be failing 🤔

File: dynamic-plugins/wrappers/backstage-community-plugin-nexus-repository-manager/src/index.ts [hunk 1]
```diff
@@ -0,0 +1 @@
+export { default } from '@backstage-community/plugin-nexus-repository-manager';
```
Comment (schultzp2020) [inline dynamic-plugins/wrappers/backstage-community-plugin-nexus-repository-manager/src/index.ts:1]:
```suggestion
export { default } from '@backstage-community/plugin-nexus-repository-manager';

```

File: dynamic-plugins/wrappers/backstage-community-plugin-nexus-repository-manager/package.json [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+    "role": "frontend-plugin",
+    "supported-versions": "1.29.2",
+    "pluginId": "nexus-repository-manager",
+    "pluginPackages": [
+      "backstage-community-plugin-nexus-repository-manager"
+    ]
+  },
+  "sideEffects": false,
+  "scripts": {
+    "build": "backstage-cli package build",
```
Comment (schultzp2020) [inline dynamic-plugins/wrappers/backstage-community-plugin-nexus-repository-manager/package.json:22]:
```suggestion
   "tsc": "tsc",
    "build": "backstage-cli package build",
```

**Pull request merged**